interface/forms/vitals/templates/vitals/vitals_textbox_conversion.tpl

```diff
@@ -0,0 +1,74 @@
+<!-- USA row comes first -->
+    {if $units_of_measurement == $MEASUREMENT_METRIC_ONLY}
+    <tr class="hide">
+    {else}
+    <tr>
+    {/if}
+    {if $units_of_measurement == $MEASUREMENT_PERSIST_IN_METRIC}
+        <td class="unfocus graph" id="{$input}">
+    {else}
+        <td class="graph" id="{$input}">
+    {/if}
+            {xlt t=$title}
+        </td>
+    {if $units_of_measurement == $MEASUREMENT_PERSIST_IN_METRIC}
+        <td class="unfocus">
+    {else}
+        <td>
+    {/if}
+            {xlt t=$unit}
+        </td>
+    {if $units_of_measurement == $MEASUREMENT_PERSIST_IN_METRIC}
+        <td class="valuesunfocus">
+    {else}
+        <td class='currentvalues p-2'>
+    {/if}
+            <input type="text" class="form-control" size='5' name='{$input}' id='{$input}_input'
+                   value="{if $vitals->$vitalsValue() != 0}{$vitals->$vitalsValue()|attr}{/if}"
+                   onChange="convUnit('usa', '{$unit}','{$input}_input')" title='{xla t=$vitalsValueUSAHelpTitle|default:''}'/>
```

Comment:
```
onChange="convUnit('usa', {$unit|attr_js},'{$input|attr}_input')" title='{xla t=$vitalsValueUSAHelpTitle|default:''}'/>
```
(note quotes removed around the attr_js call)

Reply:
an annoying thing with piping in smarty is that when you pipe a function, it actually acts on the parameter and not the function output (so default:'' will act on the $vitalsValueUSAHelpTitle parameter and not the output of the function); super odd and not sure why smarty does this.

Reply:
So I'm wondering why we don't have xla as a modifier instead of a function call?  That would solve this problem.  So it would be {$vitalsValueUSAHelpTitle|default:''|xla}.  Did this not work in the past?

Reply:
I added modifier functions so we can solve this problem.  Should now be fixed.